x-pack/plugins/ml/public/application/data_frame_analytics/common/fields.ts

```diff
@@ -187,6 +187,18 @@ export const getDefaultFieldsFromJobCaps = (
   let predictedField: string | undefined;
 
   if (isOutlierAnalysis(jobConfig.analysis)) {
+    if (jobConfig.analysis.outlier_detection.compute_feature_influence) {
+      featureImportanceFields.push({
+        id: `${resultsField}.${FEATURE_INFLUENCE}`,
+        name: `${resultsField}.${FEATURE_INFLUENCE}`,
+        type: KBN_FIELD_TYPES.UNKNOWN,
+      });
+      // remove flattened feature influence fields
+      fields = fields.filter(
+        (field: any) => !field.name.includes(`${resultsField}.${FEATURE_INFLUENCE}.`)
```

Comment:
There is `Field` type available 
```suggestion
        (field: Field) => !field.name.includes(`${resultsField}.${FEATURE_INFLUENCE}.`)
```

Reply:
Updated in af2a22dab20501c37e959f5109ac5da42e975692